app/components/game-general-info.js

```diff
@@ -33,9 +33,10 @@ class GameGeneralInfo extends React.PureComponent {
     const game = this.props.gameData;
     return (
       <Form>
-        <FormGroup controlId='formName'>
+        <FormGroup controlId='formName' validationState={this.props.validName}>
```

Comment:
dejar espacio: ={ this.props.validName } y este bloque tiene demasiada indentacion. Solo dos espacios tiene que ser